tfjs-backend-webgpu/src/base.ts

```diff
@@ -32,16 +32,7 @@ if (isWebGPUSupported()) {
 
     const adapter = await navigator.gpu.requestAdapter(gpuDescriptor);
     const deviceDescriptor: GPUDeviceDescriptor = {};
-
-    // Note that timestamp-query-inside-passes is not formally in spec as
-    // timestamp within a pass is not generally supported on all the platforms.
-    // More details can be found at
-    // https://github.com/gpuweb/gpuweb/blob/main/proposals/timestamp-query-inside-passes.md
-    if (adapter.features.has('timestamp-query-inside-passes')) {
-      deviceDescriptor.requiredFeatures =
-          // tslint:disable-next-line:no-any
-          ['timestamp-query-inside-passes' as any];
-    }
+    deviceDescriptor.requiredFeatures = ['timestamp-query'];
```

Comment:
Do we need to check whether `adapter.features.has('timestamp-query')` is true?

Reply:
yes, we need a check